release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -86,6 +86,68 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-mirror-v2_{context}"]
+==== oc-mirror plugin v2 (Technology Preview)
+
+The oc-mirror plugin v2 for {product-title} includes new features and functionalities that improve the mirroring process for Operator images and other {product-title} content.
+
+The following are the key enhancements and features in oc-mirror plugin v2:
+
+* *Automatic generation of IDMS and ITMS objects*:
++
+oc-mirror plugin v2 automatically generates a comprehensive list of `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects after each run. These objects replace the `ImageContentSourcePolicy` (ICSP) used in oc-mirror plugin v1. This enhancement eliminates the need for manual merging and cleanup of operator images and ensures all necessary images are included.
+
+* *CatalogSource objects*: 
++
+CatalogSource Objects Creation, where the plugin now generates CatalogSource objects for all relevant catalog indexes to enhance the application of oc-mirror's output artifacts to disconnected clusters.
```

Comment:
Is "CatalogSource Objects Creation" a feature name? Just checking the title case capitalization is intended

Reply:
a functionality actually, but for the new feature. I will change it.